packages/preview-middleware/src/base/flp.ts

```diff
@@ -332,20 +345,40 @@ export class FlpSandbox {
         for (const app of this.config.apps) {
             if (app.local) {
                 const manifest = JSON.parse(readFileSync(join(app.local, 'webapp/manifest.json'), 'utf-8'));
-                this.addApp(manifest, app);
+                this.addApp(manifest, app, { manifest: true });
                 this.router.use(app.target, serveStatic(join(app.local, 'webapp')));
                 this.logger.info(`Serving additional application at ${app.target} from ${app.local}`);
             }
         }
     }
 
     /**
-     * Create required routes for flex.
+     * Retrieves the configuration settings for UI5 flexibility services.
      *
-     * @returns template configuration for flex.
+     * @returns An array of flexibility service configurations, each specifying a connector
+     *          and its options, such as the layers it applies to and its service URL, if applicable.
      */
-    private createFlexHandler(): TemplateConfig['ui5']['flex'] {
-        const workspaceConnectorPath = 'open/ux/preview/client/flp/WorkspaceConnector';
+    private getFlexSettings(): TemplateConfig['ui5']['flex'] {
+        const localConnectorPath = 'open.ux.preview.client.flp.LocalConnector';
+
+        return [
+            { connector: 'LrepConnector', layers: [], url: '/sap/bc/lrep' },
+            {
+                applyConnector: localConnectorPath,
+                writeConnector: localConnectorPath,
+                custom: true
+            },
+            {
+                connector: 'LocalStorageConnector',
+                layers: ['CUSTOMER', 'USER']
+            }
+        ];
+    }
```

Comment:
Now I will always have this connector regardless of the UI5 version, right?

Reply:
Yes, but if it's not defined in the case of ui5 version 1.71 it will be skipped.

Reply:
Since there is also no way to check the UI5 version at this point in the code, it will be there regardless of the version.